src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/xunitrunner-perf/xunitrunner-perf.py

```diff
@@ -123,7 +123,7 @@ def _prepare_windows_env_for_perf(xunit_perf_drop, test_location):
 
 # used to copy the required xunit perf runner an its dependencies
 # note that the perf runner will only be present for perf tests.
-def _prepare_perf_execution_environment(settings, perf_runner, osgroup):
```

Comment:
It is still going to be used as a part of CoreCLR runs for the time being and I wanted it to be modified based on your suggestions about "osgroup" option

---
In reply to: [69825038](https://github.com/dotnet/buildtools/pull/860#discussion_r69825038) [](ancestors = 69825038)